pkg/process/monitor/process_monitor.go

```diff
@@ -0,0 +1,247 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux
+// +build linux
+
+package monitor
+
+import (
+	"regexp"
+	"runtime"
+	"sync"
+
+	"github.com/vishvananda/netlink"
+
+	"github.com/DataDog/datadog-agent/pkg/network/config"
+	"github.com/DataDog/datadog-agent/pkg/process/util"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"github.com/DataDog/gopsutil/process"
```

Comment:
```suggestion
	"github.com/DataDog/gopsutil/process"
	"github.com/vishvananda/netlink"

	"github.com/DataDog/datadog-agent/pkg/network/config"
	"github.com/DataDog/datadog-agent/pkg/process/util"
	"github.com/DataDog/datadog-agent/pkg/util/log"
```

Reply:
We don't mix repo source DataDog vs external, in all our headers, ex: pkg/network/tracer/gateway_lookup.go

Reply:
I think I miss read, didn't see the gopsutil, thanks